### DISCUSSION TITLE
is this project any different from ranger ?

### DISCUSSION THREAD

**Discussion by Greatz08** [jarun/nnn#1939] [General]
From preview what i saw same things i could do in ranger too and it doesnt come by default with previews same like ranger then what makes it special can you guys seriously tell please.
Asking here because i saw it on trending and was little surpised and excited to what this project has to offer after seeing 19k stars but just from preview and 1 min experience i am questioning like what all things are special in it when compared to ranger which is also very popular cli file manager. Since many of you might have tried both so i thought to you can explain me major things  :-))

**Comment by cerebrl** [comment]
The simplicity, for me personally, is its main feature. I chose not to use Ranger because I didn't want all the features. Plus, I liked how nnn is written mostly in C, so, along with its simplicity, it makes for a really snappy experience.

**Comment by jarun** [comment]
Please visit the project wiki which has lots of details on what you can achieve with nnn.
